it/uk/gov/hmrc/tai/integration/TaxCodeMismatchSpec.scala

```diff
@@ -33,7 +33,8 @@ class TaxCodeMismatchSpec extends IntegrationSpec {
   }
 
   val apiUrl = s"/tai/$nino/tax-account/tax-code-mismatch"
-  def request = FakeRequest(GET, apiUrl).withHeaders("X-SESSION-ID" -> generateSessionId)
+  def request = FakeRequest(GET, apiUrl)
+    .withHeaders("X-SESSION-ID" -> generateSessionId, "AUTHORIZATION" -> "Bearer 11")
```

Comment:
Use HeaderNames.xSessionId and HeaderNames.authorisation

Reply:
Updated